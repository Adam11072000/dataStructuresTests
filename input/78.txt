init 757
setLabelScore 3590 665 2670 -4
quit