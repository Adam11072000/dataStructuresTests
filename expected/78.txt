init done.
addImage: SUCCESS
setLabelScore: SUCCESS
deleteImage: FAILURE
resetLabelScore: FAILURE
setLabelScore: INVALID_INPUT
deleteImage: INVALID_INPUT
quit done.
